docs/book/overview.md

```diff
@@ -0,0 +1,43 @@
+# Overview
+
+vSphere CSI driver consists on 2 main components
+- vSphere CSI Controller
+- Syncer
+  - Metadata Syncer
+  - Full Sync
+  - CnsOperator
+
+### vSphere CSI Controller
+The vSphere Container Storage Interface (CSI) Driver provides a CSI interface used by Container Orchestrators to manage the lifecycle of vSphere volumes.
+
+CSI driver is responsible for volume provisioning, attaching and detaching the volume to VMs, mounting, formatting and unmounting volumes from the pod within the node VM etc. 
+
+### Syncer
+#### Metadata Syncer
+The syncer is responsible for pushing PV, PVC and Pod metadata to CNS. These metadata will be presented to vCenter or Storage Admin on vSphere UI
+
+#### Full Sync
+Full Sync is responsible keeping CNS upto date with all Kubernetes volume metadata PV, PVC and Pod metadata. Some cases where full sync would be helpful are listed below:
```

Comment:
Full sync is responsible for keeping CNS up to date with Kubernetes volume metadata information like PV, PVC and Pod etc.